AFQ/tasks/data.py

```diff
@@ -373,6 +414,85 @@ def csd_params(dwi, brain_mask, gtab, data,
     return csdf.shm_coeff, meta
 
 
+@immlib.calc("csd_aodf_params")
+@as_file(suffix='_model-csd_param-aodf_dwimap.nii.gz',
+         subfolder="models")
+@as_img
+def csd_aodf(csd_params, n_threads, low_mem):
+    """
+    full path to a nifti file containing
+    SSST CSD ODFs filtered by unified filtering [1]
+
+    References
+    ----------
+    [1] Poirier and Descoteaux, 2024, "A Unified Filtering Method for
+        Estimating Asymmetric Orientation Distribution Functions",
+        Neuroimage, https://doi.org/10.1016/j.neuroimage.2024.120516
+    """
+    sh_coeff = nib.load(csd_params).get_fdata()
+
+    logger.info("Applying unified filtering to CSD ODFs...")
```

Comment:
```suggestion
    logger.info("Applying unified filtering to generate asymmetric CSD ODFs...")
```

Maybe just a bit more information for users to work with, so they understand what's going on